gravity-forms/gw-progress-meter.php

```diff
@@ -190,8 +190,15 @@ public function prepare_label( $label, $type, $number ) {
 		$bits = explode( ' ', $label );
 
 		foreach ( $bits as &$bit ) {
+			// In order to support formatted numbers we needed to move away from use the `%d` placeholder. This code
+			// automatically updates existing uses of the `%d` to `%s`.
 			if ( strpos( $bit, '%d' ) !== false ) {
-				$bit = sprintf( '<span class="gwpm-%s-number">' . $bit . '</span>', $type, $number );
+				$bit = str_replace( '%d', '%s', $bit );
+			}
+			if ( strpos( $bit, '%s' ) !== false ) {
+				$currency      = new RGCurrency( GFCommon::get_currency() );
```

Comment:
Is `$currency` here just used to get the default thousand separator for the site?

Reply:
Yup. Let me know if there's a better way!